src/aat/resources/features/F-058/F-058.feature

```diff
@@ -0,0 +1,94 @@
+@F-058
+Feature: Start case creation as Citizen
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-241
+  Scenario: must start case creation process successfully for correct inputs
+    Given   a user with [an active profile in CCD]
+    When    a request is prepared with appropriate values
+    And     it is submitted to call the [Start case creation as Citizen] operation of [CCD Data Store]
+    Then    a positive response is received
+    And     the response [code is HTTP-200]
+    And     the response has all the details as expected
+
+  @S-237 @Ignore # Response code mismatch, expected: 401, actual: 403
+  Scenario: must return 401 when request provides invalid authentication credentials
+    Given   a user with [an active profile in CCD]
+    When    a request is prepared with appropriate values
+    And     the request [contains a dummy user id]
+    And     it is submitted to call the [Start case creation as Citizen] operation of [CCD Data Store]
+    Then    a negative response is received
+    And     the response [code is HTTP-401]
+    And     the response has all the details as expected
+
+  @S-237
+  Scenario: must return 403 when the request contains a dummy user id
+    Given   a user with [an active profile in CCD]
+    When    a request is prepared with appropriate values
+    And     the request [contains a dummy user id]
+    And     it is submitted to call the [Start case creation as Citizen] operation of [CCD Data Store]
+    Then    a negative response is received
+    And     the response [code is HTTP-403]
+    And     the response has all the details as expected
+
+  @S-238 @Ignore # Response code mismatch, expected: 403, actual: 200
+  Scenario: must return 403 when the request contains a jurisdiction id user is unauthorised to access
+    Given   a user with [an active profile in CCD]
+    When    a request is prepared with appropriate values
+    And     the request [contains a jurisdiction id user is unauthorised to access]
```

Comment:
contains a jurisdiction id user is unauthorised to access. It should be free flow

Reply:
sorry this one already approved by Mutlu in RDM-6320 as per his suggestion